climakitae/util/utils.py

```diff
@@ -520,3 +520,40 @@ def summary_table(data):
         df = df.sort_values(by=["year"])
 
     return df
+
+
+def convert_to_local_time(data, selections, lat, lon) -> xr.Dataset:
+    """
+    Converts the inputted data to the local time of the selection.
+    """
+    # 1. Find the other data
+    start, end = selections.time_slice
+    tz_selections = copy.copy(selections)
```

Comment:
Replace with` copy.deepcopy`

Reply:
Potential V2: sense the selections from the data, no need to pass in selections with it